README.md

```diff
@@ -127,6 +128,49 @@ After successful deployment, the installer automatically:
 2. **Validates Hypervisors**: Lists available hypervisors to confirm successful deployment
 3. **Provides Status**: Displays deployment status and service availability
 
+## Deploy Ceph
+
+JetBrew supports deploying an external Ceph cluster that can be integrated with RHOSO for block storage (Cinder), image storage (Glance), and VM disk storage (Nova). The Ceph deployment is performed separately from the main RHOSO deployment.
+
+### Prerequisites
+
+Before deploying Ceph, ensure:
+- **Sufficient Nodes**: You have enough nodes available.
+- **Registry Access**: Access to Red Hat registry (`registry.redhat.io`) with valid credentials.
+- **rhos release rpm**: Make sure You have Redhat openstack rpm. 
+
+### Configuration
+
+#### Required Variables
+
+Edit the `ansible/group_vars/all.yml` file and configure the following mandatory variables:
```

Comment:
Add about using custom inventory for ceph. or if using default inventory, give a note saying that last few nodes will be consumed based on the ceph node count.